packages/react/src/__test__/AsyncBoundary.spec.tsx

```diff
@@ -1,10 +1,17 @@
 import { render } from '@testing-library/react'
+import { AsyncBoundary } from '../AsyncBoundary'
+import { FALLBACK, TEXT } from './utils'
 
-describe('test', () => {
-  it('should test', () => {
-    const TEST_TEXT = 'suspensive'
-    const fixture = render(<div>{TEST_TEXT}</div>)
+describe('AsyncBoundary', () => {
+  it('should show children', () => {
+    const rendered = render(<AsyncBoundary rejectedFallback={FALLBACK}>{TEXT}</AsyncBoundary>)
+    expect(rendered.getByText(TEXT)).toBeInTheDocument()
+  })
+})
 
-    expect(fixture.getByText(TEST_TEXT)).toBeInTheDocument()
+describe('AsyncBoundary.CSROnly', () => {
```

Comment:
I think it don't need to express it because renderer itself is making component mount. 

If you think the test code becomes a document(specification), why don't you write the description in more detail?

Reply:
I left suggestion! 😄

Reply:
Cool!